storage/innobase/xtrabackup/test/t/xbcloud.sh

```diff
@@ -165,8 +190,105 @@ if  grep -q "Upload failed" $topdir/pxb-2279.log ; then
     die 'xbcloud exit with error on upload'
 fi
 
+# PXB-2112 - Support for storage class
```

Comment:
2112??

Reply:
test case for PXB-2112 was not ported to 2.4 but it is present in 8.0. so just ported it back.